sections/semantics-document-metadata.include

```diff
@@ -1707,14 +1707,22 @@
   available to script once the <a>event loop</a> reaches its <i>update the rendering</i> step.
 
   A style sheet in the context of the {{Document}} of an <a>HTML parser</a> or <a>XML parser</a> is
-  said to be <dfn>a style sheet that is blocking scripts</dfn> if the element was created by that
-  {{Document}}'s parser, and the element is either a <{style}> element or a <{link}> element that
-  was an <a href="#link-type-stylesheet">external resource link</a> when the element was created by
-  the parser, and the element's style sheet was enabled when the element was created by the parser,
-  and the element's <a>style sheet ready</a> flag is not yet set, and, the last time the
-  <a>event loop</a> reached step 1, the element was in that {{Document}}, and the user
-  agent hasn't given up on that particular style sheet yet. A user agent may give up on a style
-  sheet at any time.
+  said to be <dfn>a style sheet that is blocking scripts</dfn> if the  the style sheet was added
+  by either
+
+    * a <{style}> element created by that {{Document}}'s parser, or
+    * a <{link}> element whose <{link/media}> attribute <a>matches the environment</a> that
+    was created by that {{Document}}'s parser as an
```

Comment:
Bikeshed complains about line-breaks in a single list item. Please remove this line-break :S